.github/workflows/update-wpt.yml

```diff
@@ -0,0 +1,70 @@
+name: WPT update
+
+on:
+  schedule:
+    # Run once a week at 12:00 AM UTC on Sunday.
+    - cron: 0 0 * * *
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  wpt-subsystem-update:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        subsystem:
+          - url
+
+    steps:
+      - uses: actions/checkout@6d193bf28034eafb982f37bd894289fe649468fc  # v4.1.7
+        with:
+          persist-credentials: false
+
+      - name: Update WPT for subsystem ${{ matrix.subsystem }}
+        run: |
+          git node wpt update --subsystem=${{ matrix.subsystem }}
```

Comment:
Where is this git extension coming from? The one we have is 

```
git node wpt ${{ matrix.subsystem }}
```

And FWIW I don't think this action install `@node-core/utils` globally to have that available.

Reply:
ı applied your suggestion : `git node wpt ${{ matrix.subsystem }}`